crates/nu-command/src/debug/info.rs

```diff
@@ -34,25 +47,186 @@ impl Command for DebugInfo {
+        match column {
+            "threadid" => Ok(Value::int(get_thread_id(), self.span)),
+            "pid" => Ok(Value::int(pid.as_u32() as i64, self.span)),
+            "ppid" => {
+                // only get information requested
+                let system_opt = SystemOpt::from((system_option, || {
+                    RefreshKind::new().with_processes(
+                        ProcessRefreshKind::new()
+                            .without_cpu()
+                            .without_disk_usage()
+                            .without_user(),
+                    )
+                }));
+
+                let system = system_opt.get_system();
+                // get the process information for the nushell pid
+                let pinfo = system.process(pid);
+
+                Ok(pinfo
+                    .and_then(|p| p.parent())
+                    .map(|p| Value::int(p.as_u32() as i64, self.span))
+                    .unwrap_or(Value::nothing(self.span)))
+            }
+            "system" => {
+                // only get information requested
+                let system_opt =
+                    SystemOpt::from((system_option, || RefreshKind::new().with_memory()));
+
+                let system = system_opt.get_system();
+
+                Ok(Value::record(
+                    record! {
+                        "total_memory" => Value::filesize(system.total_memory() as i64, self.span),
+                        "free_memory" => Value::filesize(system.free_memory() as i64, self.span),
+                        "used_memory" => Value::filesize(system.used_memory() as i64, self.span),
+                        "available_memory" => Value::filesize(system.available_memory() as i64, self.span),
+                    },
+                    self.span,
+                ))
+            }
+            "process" => {
+                // only get information requested
+                let system_opt = SystemOpt::from((system_option, || {
+                    RefreshKind::new().with_processes(
+                        ProcessRefreshKind::new()
+                            .without_cpu()
+                            .without_disk_usage()
+                            .without_user(),
+                    )
+                }));
+
+                let system = system_opt.get_system();
+                // get the process information for the nushell pid
+                let pinfo = system.process(pid);
+
+                if let Some(p) = pinfo {
+                    Ok(Value::record(
+                        record! {
+                            "memory" => Value::filesize(p.memory() as i64, self.span),
+                            "virtual_memory" => Value::filesize(p.virtual_memory() as i64, self.span),
+                            "status" => Value::string(p.status().to_string(), self.span),
+                            "root" => Value::string(p.root().to_string_lossy().to_string(), self.span),
```

Comment:
We cannot use `root()` as is right now; not until there's a new version `sysinfo` crate release. This is why I have the changes above getting "root" differently, with the comments.

Reply:
Nice catch, missed this while copy-pasting !